src/Controller/CRUDController.php

```diff
@@ -1563,6 +1569,26 @@ private function checkParentChildAssociation(Request $request, object $object):
         }
     }
 
+    /**
+     * Checks whether $parentObject is equal to $childParentValue or part of it.
+     */
+    private static function equalsOrInList($needle, $haystack): bool
```

Comment:
please add type hints here, or at least add phpdoc. Why is the method static?

Reply:
The method is static because it doesn't need `$this`.

Reply:
The type hints have been added.